plotly-plot.html

```diff
@@ -221,247 +225,11 @@
-       *  Polymer element.
-       *
-       * @see the {@link https://plot.ly/javascript/plotlyjs-function-reference/|plotly.js function reference}
-       */
-      addTraces: function (traces, traceIndices) {
-        var self = this;
-        return Plotly.addTraces(self.$.plot, traces, traceIndices)
-          .then(function (plotDiv) {
-            // Update the polymer properties to reflect the updated data
-            self.data = plotDiv.data;
-            return self;
-          });
-      },
-
-      /**
-       * Delete the specified traces from the plot.
-       *
-       * @param {(number|Array<number>)=} traceIndices
-       *  an individual index or an array of indices specifying which traces to
-       *  delete
-       *
-       * @return {Promise<Polymer.Base>}
-       *  a Promise for the asynchronous plot update that resolves to the
-       *  Polymer element.
-
-       * @see the {@link https://plot.ly/javascript/plotlyjs-function-reference/|plotly.js function reference}
-       */
-      deleteTraces: function (traceIndices) {
-        var self = this;
-        return Plotly.deleteTraces(self.$.plot, traceIndices)
-          .then(function (plotDiv) {
-            // Update the polymer properties to reflect the updated data
-            self.data = plotDiv.data;
-
-            return self;
-          });
-      },
-
-      /**
-       * Move a specified set traces from the plot to a newly specified set of
-       * destination trace positions.
-       *
-       * @param {(number|Array<number>)=} traceIndicesFrom
-       *  an individual index or an array of indices specifying which traces to
-       *  move
-       * @param {(number|Array<number>)=} traceIndicesTo
-       *  an individual index or an array of indices specifying where the
-       *  traces should move
-       *
-       * @return {Promise<Polymer.Base>}
-       *  a Promise for the asynchronous plot update that resolves to the
-       *  Polymer element.
-       *
-       * @see the {@link https://plot.ly/javascript/plotlyjs-function-reference/|plotly.js function reference}
-       */
-      moveTraces: function (traceIndicesFrom, traceIndicesTo) {
-        var self = this;
-
-        return Plotly.moveTraces(self.$.plot, traceIndicesFrom, traceIndicesTo)
-          .then(function (plotDiv) {
-            // Update the polymer properties to reflect the updated data
-            self.data = plotDiv.data;
-
-            return self;
-          });
-      },
-
-      /**
-       * Clear all plots and snapshots.
-       *
-       * @return {Polymer.Base} the current element
-       *
-       * @see the {@link https://plot.ly/javascript/plotlyjs-function-reference/|plotly.js function reference}
-       */
-      purge: function () {
-        Plotly.purge(this.$.plot);
-
-        this.$.plot.data = [];
-        this.data = [];
-
-        this.$.plot.layout = {};
-        this.layout = {};
-
-        return this;
+      registered: function() {
```

Comment:
Also, if you add a function, please add JSDoc for it, as every other function here has.